scripts/skills/actives/rf_onslaught_skill.nut

```diff
@@ -36,7 +36,7 @@ this.rf_onslaught_skill <- ::inherit("scripts/skills/skill", {
 			id = 20,
 			type = "text",
 			icon = "ui/icons/warning.png",
-			text = ::Reforged.Mod.Tooltips.parseString("Does not affect allies who are [fleeing,|Concept.Morale] [$ $,|Skill+stunned_effect] or [$ $|Skill+sleeping_effect]")
+			text = ::Reforged.Mod.Tooltips.parseString("Does not affect allies who are [fleeing|Concept.Morale], [$ $|Skill+stunned_effect], or [$ $|Skill+sleeping_effect]")
```

Comment:
The second comma is redundant as its right before an `or`.
That was already an issue before but we could use this PR to fix it